contracts/upgradeable_contracts/InterestReceiverStakeBuyback.sol

```diff
@@ -33,7 +33,15 @@ contract InterestReceiverStakeBuyback is IInterestReceiver, Ownable, Claimable {
         // (min received %) * (amount / 1 DAI) * (STAKE per 1 DAI)
         uint256 minAmount = (minReceivedFraction * amount * uniswapRouterV2.getAmountsOut(1 ether, path)[2]) / 10**36;
 
-        uniswapRouterV2.swapExactTokensForTokens(amount, minAmount, path, burnAddress, now);
+        bytes memory data = abi.encodeWithSelector(
+            uniswapRouterV2.swapExactTokensForTokens.selector,
+            amount,
+            minAmount,
+            path,
+            burnAddress,
+            now
+        );
+        address(uniswapRouterV2).call(data);
```

Comment:
Should we receive the returned by `call` values but ignore them?

Reply:
Why do we need to do so? Solidity compiler will just generate redundant bytecode for that (`returndatasize`/`returndatacopy` opcodes).

Reply:
OK. If so, it is OK for me.